variables.tf

```
@@ -4,6 +4,8 @@ variable "log_analytics_workspace" {
     name                = string
     resource_group_name = string
     location            = string
+    workspace_id        = string
+    primary_shared_key  = string
   })
   description = "Log Analytics Worksapce that all VMs are connected to for monitoring"
   nullable = false
```

```suggestion
  nullable = false
  sensitive = true
```
as we now have a secret inside of this object, we should make sure that it is sensitive